etstool.py

```diff
@@ -243,28 +261,28 @@ def install(runtime, toolkit, environment, editable, source):
         | extra_dependencies["test"]
         | extra_dependencies["examples"]
         | extra_dependencies["editors"]
+        | runtime_dependencies[runtime]
         | ci_dependencies
     )
 
     # edm commands to setup the development environment
     commands = [
-        "edm environments create {environment} --force --version={runtime}"
+        "edm -c {config} environments create {environment} --force --version={runtime}",
+        "edm -c {config} install -y -e {environment} " + " ".join(packages),
     ]
 
-    commands.extend(
-        [
-            "edm install -y -e {environment} " + " ".join(packages),
-            "edm run -e {environment} -- pip install --force-reinstall -r ci-src-requirements.txt --no-dependencies",
-            "edm run -e {environment} -- python setup.py clean --all",
-        ]
-    )
-
-    # pip install pyqt5, pyqt6, pyside2 and pyside6, because we don't have them
-    # in EDM yet
-    if toolkit == 'pyside2':
-        commands.append("edm run -e {environment} -- pip install pyside2")
-    elif toolkit == 'pyqt6':
-        commands.append("edm run -e {environment} -- pip install pyqt6")
+    if (runtime, toolkit) in edm_versions:
+        commands.append(
+            "edm -c {config} install -y -e {environment} {toolkit}"
+        )
+    elif toolkit == 'wx':
+        if sys.platform != 'linux':
+            commands.append("edm -c {config} run -e {environment} -- pip install wxPython")
+        else:
+            # XXX this is mainly for TravisCI workers; need a generic solution
+            commands.append(
+                "edm -c {config} run -e {environment} -- pip install -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-16.04/ wxPython"
```

Comment:
Hmm; 16.04 is rather old now; I'm surprised this is still working.

Reply:
I've added an issue #1970